src/Nox.Lib/Middlewares/NoxExceptionHanderMiddleware.cs

```diff
@@ -31,7 +30,7 @@ public async Task InvokeAsync(HttpContext httpContext)
         {
             await HandleTypeValidationExceptionAsync(httpContext, ex);
         }
-        catch(ConcurrencyException ex)
+        catch (ConcurrencyException ex)
         {
             await HandleConcurrencyExceptionAsync(httpContext, ex);
```

Comment:
this should be the status code if ETag is outdated 

//
// Summary:
//     Equivalent to HTTP status 412. System.Net.HttpStatusCode.PreconditionFailed indicates
//     that a condition set for this request failed, and the request cannot be carried
//     out. Conditions are set with conditional request headers like If-Match, If-None-Match,
//     or If-Unmodified-Since.
PreconditionFailed = 412,

Reply:
we can improve the middleware, avoid all this if, and adding some metadata to expected exceptions, see my comment above?